### DISCUSSION TITLE
Failed to build gem native extension: Mac OS Ventura (13) on x86_64

### DISCUSSION THREAD

**Discussion by delphaber** [jeremyevans/sequel_pg#53] [General]
I'm opening a discussion like suggested in https://github.com/jeremyevans/sequel_pg/issues/52

I've found that this commit ( https://github.com/jeremyevans/sequel_pg/commit/4748b1dc9290a6ba13e1ce8fbae4969fff6029cd ) contains a command that worked for me:

`gem install sequel_pg -- --with-ldflags="-Wl,-U,_pg_get_pgconn -Wl,-U,_pg_get_result_enc_idx -Wl,-U,_pgresult_get -Wl,-U,_pgresult_stream_any"` 🥳 

Is it possible that this fix ( https://github.com/jeremyevans/sequel_pg/commit/d6e1081280644dbfa4c0a8a2af129d2ab2e7c465 ) does not work because I'm on Intel Chip so my platform is `x86_64-darwin-22` and not something like `/arm64-darwin/` ?

**Comment by jeremyevans** [comment]
Could be.  You could try changing it to be just `RUBY_PLATFORM =~ /darwin/` and see if that works.  If that works for you, please submit a pull request for it.

**Reply by delphaber** [reply]
I'll do!

**Reply by delphaber** [reply]
It worked :)

I've created a PR here: https://github.com/jeremyevans/sequel_pg/pull/54
